README.md

```diff
@@ -17,38 +17,50 @@ The command should run with "-c filePath" to load the config file and the config
 
 Config file example:
 ```
-endpoint = "sp.gnfd.cc"
-grpcAddr = "greenfield.bnbchain.world:9090"
-chainId = "greenfield_9000-1741"
+# the primary storage provider endpoint
+endpoint = "https://gnfd-testnet-sp-1.nodereal.io"
+# the grpc address of greenfield
+grpcAddr = "gnfd-testnet-fullnode-cosmos-us.bnbchain.org:9090"
+# the chain id info of greenfield
+chainId = "greenfield_5600-1"
 privateKey = "ec9577ceafbfa462d510e505df63aba8f8b23886fefxxxxxxxxxxxxx"
 ```
 
 ### support commands
 
 ```
 COMMANDS:
-   mb             create bucket
-   update-bucket  update bucket meta on chain
-   put            upload an object
-   get            download an object
-   create-obj     create an object
-   get-hash       compute hash roots of object
-   del-obj        delete an existed object
-   del-bucket     delete an existed bucket
-   head-obj       query object info
-   head-bucket    query bucket info
-   challenge      Send challenge request
-   list-sp        list sp info
-   mg             create group
-   update-group   update group member
-   head-group     query group info
-   head-member    check group member if it exists
-   del-group      delete an existed group
-   buy-quota      update bucket meta on chain
-   get-price      get the quota price of sp
-   quota-info     get quota info of the bucket
-   ls-bucket      list bucket info of the provided user
-   ls             list object info of the bucket
+   mb                      create bucket
+   update-bucket           update bucket meta on chain
```

Comment:
Can these commands be unified? Some are abbreviated, some are not. E.g `mb`,`update-bucket`

Reply:
yes,  Naming can be done without abbreviations

Reply:
updated